apps/browser/qml/pages/components/BookmarkItem.qml

```diff
@@ -111,6 +111,9 @@ ListItem {
                 onClicked: {
```

Comment:
There is the previous redefinition of `sailfish_browser-me-edit` just above this line (number 109 at the time of writing), which says "Edit" instead of "Edit bookmark".

Another "Edit" with the same ID is defined here:
https://github.com/sailfishos/sailfish-browser/blob/3ed8aa667d13fd0820706e7782bb610877acf0aa/apps/browser/qml/pages/components/FavoriteContextMenu.qml#L48

Reply:
Thanks for spotting this. I've removed the duplication and tried to make things a bit clearer.